src/Sfa.Roatp.Indexer.Infrastructure/Settings/ElasticsearchSettings.cs

```diff
@@ -20,7 +21,7 @@ public ElasticsearchSettings(IProvideSettings settingsProvider)
         private string[] GetSetting(string configName)
```

Comment:
This can be IEnumerable<string> and then keep the System.Collections.Generic using and remove the System.Configuration